[2024-05-20T15:42:15.049+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-20T15:42:15.057+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-20T15:42:13.451174+00:00 [queued]>
[2024-05-20T15:42:15.060+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-20T15:42:13.451174+00:00 [queued]>
[2024-05-20T15:42:15.060+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-05-20T15:42:15.065+0000] {taskinstance.py:2330} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-05-20 15:42:13.451174+00:00
[2024-05-20T15:42:15.068+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3888) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-20T15:42:15.069+0000] {standard_task_runner.py:63} INFO - Started process 3890 to run task
[2024-05-20T15:42:15.070+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-05-20T15:42:13.451174+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp0ycs21du']
[2024-05-20T15:42:15.070+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask is_weather_api_ready
[2024-05-20T15:42:15.091+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-05-20T15:42:13.451174+00:00 [running]> on host 39024998a807
[2024-05-20T15:42:15.130+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T15:42:13.451174+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T15:42:13.451174+00:00'
[2024-05-20T15:42:15.131+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-20T15:42:15.131+0000] {baseoperator.py:399} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-05-20T15:42:15.132+0000] {http.py:137} INFO - Poking: /data/2.5/weather?q=Pune&appid=f3ab10bd45b1669434c4da7f043b1a5a
[2024-05-20T15:42:15.136+0000] {base.py:84} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-05-20T15:42:15.457+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-20T15:42:15.464+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-20T15:42:15.475+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=is_weather_api_ready, run_id=manual__2024-05-20T15:42:13.451174+00:00, execution_date=20240520T154213, start_date=20240520T154215, end_date=20240520T154215
[2024-05-20T15:42:15.504+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-20T15:42:15.546+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296: AirflowProviderDeprecationWarning: Call to deprecated class SimpleHttpOperator. (Class `SimpleHttpOperator` is deprecated and will be removed in a future release. Please use `HttpOperator` instead.)
  result = cls.__new__(cls)

[2024-05-20T15:42:15.562+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-20T15:42:15.563+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
